src/css/style.scss

```diff
@@ -308,7 +294,9 @@ table {
   padding-left: 0;
   font-size: 1em;
 }
-
+/* This controls the friends list on the left-sidebar.
+As of right now, I can't see any other use for it.
+Likely needs its name changed.*/
 .historical-list-listItem {
```

Comment:
For sure agree with you on the name change here Dougie - the weekly calendar also uses the word `historical` for class names, and I think that one could be more specific too. in any case there needs to be some specialization between the two.